tenant_schemas/utils.py

```diff
@@ -12,30 +12,75 @@
 from django.core import mail
 
 
+def all_tenant_enabled_databases():
+    return [db for db, config in settings.DATABASES.items()
+            if config['ENGINE'] == 'tenant_schemas.postgresql_backend' ]
```

Comment:
I would change this to use `connections` and check if each connection is an instance of `tenant_schemas.postgresql_backend`, since someone can override it.